sdk/eventgrid/azure-eventgrid/samples/async_samples/sample_publish_events_to_a_topic_with_signature_async.py

```diff
@@ -0,0 +1,45 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+"""
+FILE: sample_publish_custom_events_to_a_topic_with_signature_async.py
+DESCRIPTION:
+    These samples demonstrate sending an EventGrid Event using a shared access signature for authentication.
+USAGE:
+    python sample_publish_custom_events_to_a_topic_with_signature_async.py
+    Set the environment variables with your own values before running the sample:
+    1) EVENTGRID_SAS - The access key of your eventgrid account.
+    2) EG_TOPIC_HOSTNAME - The topic hostname. Typically it exists in the format
+    "<YOUR-TOPIC-NAME>.<REGION-NAME>.eventgrid.azure.net".
+"""
+import os
+import asyncio
+from azure.eventgrid import EventGridEvent, generate_sas
```

Comment:
```suggestion
from azure.eventgrid import EventGridEvent
```
In general, it looks like the publish using sas samples don't use the `generate_sas` method in EG and use `AzureSasCredential`. I would think that this sample would use the `generate_sas` provided by EG instead, so it might be more clear to name the sample something like `sample_publish_events_to_a_topic_with_azure_sas_credential_async.py`? --> same comment for the sync version

Reply:
good suggestion - yes - it's unlikey that the same user uses AzureSasCrednetial and generates a sas by themselves.